tags/tags-impl/impl/pom.xml

```diff
@@ -0,0 +1,132 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <name>tags-impl</name>
+  <groupId>org.sakaiproject.tags</groupId>
+  <artifactId>tags-impl</artifactId>
+  
+  <packaging>jar</packaging>
+  
+  <parent>
+    <artifactId>tags-base</artifactId>
+    <groupId>org.sakaiproject.tags</groupId>
+    <version>12-SNAPSHOT</version>
+    <relativePath>../../pom.xml</relativePath>
+  </parent>
+
+  <properties>
+    <deploy.target/>
+  </properties>
+
+
+  <build>
+    <resources>
+      <resource>
+        <directory>${basedir}/src/resources</directory>
+      </resource>
+    </resources>
+  </build>
+
+  <dependencies>
+    
+    <dependency>
+      <groupId>com.googlecode.json-simple</groupId>
```

Comment:
The version numbers and dependencies here should be defined in dependencyManagement in master and some may already be defined.